.github/workflows/unittests.yml

```
@@ -10,26 +10,20 @@ jobs:
   TFLite:
     runs-on: ubuntu-latest
     if: "!contains(github.event.head_commit.message, 'ci-skip')"
-    container:
-      image: tensorflow/tensorflow:2.2.0-custom-op-ubuntu16
-
     steps:
       - uses: actions/checkout@v1
         with:
           submodules: true
-      - name: Install Bazelisk
-        run: |
-          curl -L https://github.com/bazelbuild/bazelisk/releases/download/v1.7.4/bazelisk-linux-amd64 > bazelisk
-          chmod +x bazelisk
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
       - name: Configure Bazel
         run: ./configure.py
         shell: bash
-        env:
-          MANYLINUX2010: 1
-        # CC unit tests needs to run after the configure is finished
-        # since the CC builds rely on compiler options generated by configure.
-      - name: Run TFLite Ops and Core CC Unit Tests
-        run: ./bazelisk test larq_compute_engine/tests:cc_tests --copt=-O2 --distinct_host_configuration=false --test_output=all
+      - name: Install pip dependencies
+        run: pip install numpy --no-cache-dir
```

I think we also used to also need `six` for TF to build properly. Not sure if this is still the case.
```suggestion
        run: pip install numpy six --no-cache-dir
```

---

Thanks for the tip, lets see what CI does.